source/command-monitoring/tests/unified/deleteMany.yml

```diff
@@ -0,0 +1,72 @@
+description: "deleteMany"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client client
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+        - commandFailedEvent
+  - database:
+      id: &database database
+      client: *client
+      databaseName: &databaseName command-monitoring-tests
+  - collection:
+      id: &collection collection
+      database: *database
+      collectionName: &collectionName test
+
+initialData:
+  - collectionName: *collectionName
+    databaseName: *databaseName
+    documents:
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+
+tests:
+  - description: "A successful deleteMany"
+    operations:
+      - name: deleteMany
+        object: *collection
+        arguments:
+          filter: { _id: { $gt: 1 } }
+    expectEvents:
+      - client: *client
+        events:
+          - commandStartedEvent:
+              command:
+                delete: *collectionName
+                deletes:
+                  - { q: { _id: { $gt: 1 }}, limit: 0 }
+                ordered: true
+              commandName: delete
+              databaseName: *databaseName
+          - commandSucceededEvent:
+              reply: { ok: 1, n: 2 }
+              commandName: delete
+
+  - description: "A successful deleteMany with write errors"
```

Comment:
this test doesn't make quite the same assertions about the reply as the legacy runner asked for:
* in command succeeded event's reply, errorCode exists
*in command succeeded event's reply,  errmsg exists and is non-empty

given that the data ends up nested in the `writeErrors` field it didn't seem that I could easily assert on it without exhaustively listing out the fields in each document in `writeErrors` (since they are not top-level). 
I also couldn't have asserted on a string being non-empty anyway, I think?
it seemed reasonable enough to me to just check that the operation throws a server-side error, and rely on the server to include errorCode and errmsg in that error, but let me know what you think. 

(same applies for the other "a successful x with write errors" tests)

Reply:
> I suppose we could assert via expectError but I'm not sure that would help us verify that the command failed event contains the expected info too.

Correct. That would be more a test of CRUD than the APM spec. I'm OK with what you have here and just asserting that `writeErrors` is an array.